solr/contrib/ltr/src/test/org/apache/solr/ltr/TestLTROnSolrCloud.java

```diff
@@ -232,58 +249,113 @@ private void createCollection(String name, String config, int numShards, int num
     solrCluster.waitForActiveCollection(name, numShards, numShards * numReplicas);
   }
 
-
   void indexDocument(String collection, String id, String title, String description, int popularity)
-    throws Exception{
+      throws Exception {
     SolrInputDocument doc = new SolrInputDocument();
     doc.setField("id", id);
     doc.setField("title", title);
     doc.setField("description", description);
     doc.setField("popularity", popularity);
+    if (popularity != 1) {
+      // check that empty values will be read as default
+      doc.setField("dvIntField", popularity);
+      doc.setField("dvLongField", popularity);
+      doc.setField("dvFloatField", ((float) popularity) / 10);
+      doc.setField("dvDoubleField", ((double) popularity) / 10);
+      doc.setField("dvStrNumField", popularity);
+      doc.setField("dvStrBoolField", popularity % 2 == 0 ? "T" : "F");
+    }
```

Comment:
I looked at `TestLTROnSolrCloud` changes today, this approach here is very elegant, combined with the `resultN_features` and `schema.xml` change.